### PR TITLE
WRR-12061: Fixed typos and JSDocs

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] I have run automated testing and it is passed
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fixed typos and JSDocs

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fixed typos
Removed params from JSDoc that were no longer used and added params that were missing.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Added the description for `afterTest` params from https://webdriver.io/docs/configurationfile/

### Links
[//]: # (Related issues, references)
WRR-12061

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: ui/wdio.conf.js [hunk 1]
```diff
@@ -19,15 +19,17 @@ exports.config = configure({
...
 	 */
 	onPrepare: function () {
 		return buildApps('ui');
 	},
 	/**
 	 * Function to be executed after a test (in Mocha/Jasmine) or a step (in Cucumber) starts.
-	 * @param {Object} test test details
+	 * @param {Object} testCase test details
+	 * @param {Object} _context scope object the test was executed with (unused)
+	 * @param {Object} result test result
```
Comment (seunghoh) [inline ui/wdio.conf.js:30]: Is this for the unnamed object param for {duration, passed}? Do we have the same case or is it common to be used normally?

File: ui/wdio.conf.js [hunk 2]
```diff
@@ -19,15 +19,17 @@ exports.config = configure({
...
+	 * @param {Number} result.duration duration of test
+	 * @param {Boolean} result.passed true if test has passed, otherwise false
```
Comment (seunghoh) [inline ui/wdio.conf.js:32]: @paul-beldean-lgp @daniel-stoian-lgp
Is this for the unnamed object param for {duration, passed}? Do we have the same case or is it common to be used normally?

Comment (paul-beldean-lgp) [reply ui/wdio.conf.js:32]: @seunghoh 
Yes and I added the description for the params from https://webdriver.io/docs/configurationfile/ for `afterTest`

**Pull request merged**